packages/api/test/pin.spec.js

```diff
@@ -541,10 +608,98 @@ describe('Pinning APIs endpoints', () => {
       assert.strictEqual(data.requestId, requestId.toString())
       assert.strictEqual(data.pin.name, 'reportdoc.pdf')
     })
+
+    it('error if user not authorised to pin', async () => {
+      token = await getTestJWT()
+      const res = await fetch(new URL('pins', endpoint).toString(), {
+        method: 'POST',
+        headers: {
+          Authorization: `Bearer ${token}`,
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          cid: 'bafybeibqmrg5e5bwhx2ny4kfcjx2mm3ohh2cd4i54wlygquwx7zbgwqs4e'
+        })
+      })
+
+      assert(!res.ok)
+      assert.strictEqual(res.status, (new PinningNotAuthorisedError()).status)
+    })
+  })
+
+  describe('POST /pins/:requestId', () => {
```

Comment:
I think this suite is somehow duplicated can you please double-check? (see [line](https://github.com/web3-storage/web3.storage/pull/705/files#diff-14c9b706d6ea2d7b7b4b2742105bb323beb6b3fdb23b862926505fcf482eb37aR835) )

Reply:
Yep, I have refactored and removed duplicates.